docs/planning/backend/app/scrapers/fragrantica_scraper.py

```
@@ -0,0 +1,312 @@
+        """Make HTTP request and return parsed HTML."""
+        self._wait_for_rate_limit()
+
+        try:
+            with httpx.Client(headers=self.HEADERS, follow_redirects=True, timeout=30.0) as client:
+                response = client.get(url)
+
+                if response.status_code != 200:
+                    return None
+
+                return BeautifulSoup(response.text, 'lxml')
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)